templates/quickstart/foundry/factory/README.md

```diff
@@ -45,10 +45,28 @@ $ forge snapshot
 $ anvil
 ```
 
+### Private key setup
+
+It's recommended to use Foundry keystore to store your development account private key.
+
+1. Export your account private key.
+2. Create a keystore and import your private key by running:
+
+```shell
+cast wallet import myKeystore --interactive
+# enter your PK when prompted and provide a password
+```
+
+> Note that `myKeystore` can be any name.
+
+This will return an address (keystore address), **copy it for later use**.
+
+3. When running `cast` commands that require a private key, use `--account myKeystore --sender <KEYSTORE_ADDRESS>` instead of `--private-key <PRIVATE_KEY>`. This will require you to enter the keystore password you provided before.
```

Comment:
No need to mention `--private-key`
```suggestion
3. When running `cast` commands that require a private key, use `--account myKeystore --sender <KEYSTORE_ADDRESS>``. This will require you to enter the keystore password you provided before.
```